node/core/approval-voting/src/lib.rs

```diff
@@ -856,6 +868,30 @@ where
 				}
 
 				actions
+			},
+			(block_hash, validator_index, candidate_hash) = sign_approvals_timers.timers.select_next_some() => {
+				gum::debug!(
+					target: LOG_TARGET,
+					?candidate_hash,
+					"Sign approval for candidate",
```

Comment:
This log should print out all that candidates that will be signed.

```suggestion
					"Sign approval for multiple candidates",
```